Description: Upstream changes introduced in version 2010.12~rc2-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 u-boot (2010.12~rc2-1) unstable; urgency=low
 .
   * New upstream release candidate.
   * Add openrd_base target.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <clint@gnu.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- u-boot-2010.12~rc2.orig/snapshot.commit
+++ u-boot-2010.12~rc2/snapshot.commit
@@ -1 +1 @@
-d0d2271f8c62e830ec65fb069b6d12dd708e3e2f  Sun, 28 Nov 2010 19:48:11 +0100
+$Format:%H  %cD$
